pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -325,6 +404,107 @@ func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextM
 	return kmaster, nil
 }
 
+func DefaultOpenAPIConfig() *openapicommon.Config {
+	return &openapicommon.Config{
+		Definitions:    openapigenerated.OpenAPIDefinitions,
+		IgnorePrefixes: []string{"/swaggerapi", "/healthz", "/controllers", "/metrics", "/version/openshift"},
+		GetOperationIDAndTags: func(servePath string, r *restful.Route) (string, []string, error) {
+			op := r.Operation
+			path := r.Path
+			//TODO/REBASE this is gross
+			if strings.HasPrefix(path, "/oapi/v1/namespaces/{namespace}/processedtemplates") {
+				op = "createNamespacedProcessedTemplate"
+			} else if strings.HasPrefix(path, "/oapi/v1/processedtemplates") {
+				op = "createProcessedTemplateForAllNamespaces"
+			} else if strings.HasPrefix(path, "/oapi/v1/namespaces/{namespace}/generatedeploymentconfigs") {
+				op = "generateNamespacedDeploymentConfig"
+			}
+			if op != r.Operation {
+				return op, []string{}, nil
+			}
+			return apiserveropenapi.GetOperationIDAndTags(servePath, r)
+		},
+		Info: &spec.Info{
+			InfoProps: spec.InfoProps{
+				Title:   "OpenShift API (with Kubernetes)",
+				Version: version.Get().String(),
```

Comment:
this is returning the kubernetes version. previously, it was returning the openshift version.